<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefPlaylist</class>
 <widget class="QWidget" name="PrefPlaylist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>367</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="playlist_tab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>&amp;Playlist</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QCheckBox" name="auto_add_to_playlist_check">
         <property name="text">
          <string>&amp;Automatically add files to playlist</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="add_consecutive_files_check">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Add &amp;consecutive files</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="remember_current_file_check">
         <property name="text">
          <string>Re&amp;member current file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="play_from_start_check">
         <property name="text">
          <string>P&amp;lay files from start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="recursive_check">
         <property name="text">
          <string>Add files in directories &amp;recursively</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="getinfo_check">
         <property name="text">
          <string>Get &amp;info automatically about files added (slow)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autosave_on_exit_check">
         <property name="text">
          <string>&amp;Save copy of playlist on exit</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>354</width>
           <height>61</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>playlist_tab</tabstop>
  <tabstop>auto_add_to_playlist_check</tabstop>
  <tabstop>add_consecutive_files_check</tabstop>
  <tabstop>remember_current_file_check</tabstop>
  <tabstop>play_from_start_check</tabstop>
  <tabstop>recursive_check</tabstop>
  <tabstop>getinfo_check</tabstop>
  <tabstop>autosave_on_exit_check</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>auto_add_to_playlist_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>add_consecutive_files_check</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
